### PR TITLE
ci: add nop versions of all scripts to our integration tests

### PR DESCRIPTION
**Pull request by behrmann**
This is to investigate @NekkoDroid's comment in https://github.com/systemd/mkosi/pull/2905#issuecomment-2260984613

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -162,6 +162,18 @@ jobs:
         DefaultDeviceTimeoutSec=180
         EOF
 
+        tee mkosi.configure <<EOF
+        echo "Hello from inside mkosi.configure!" >&2
+        cat
+        EOF
+
+        for script in sync prepare buuild postinst finalize postoutput clean
```
Comment (DaanDeMeyer) [inline .github/workflows/ci.yml:170]: typo

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -162,6 +162,19 @@ jobs:
         DefaultDeviceTimeoutSec=180
         EOF
 
+        tee mkosi.configure <<EOF
+        echo "Hello from inside mkosi.configure!" >&2
+        cat
+        EOF
+
+        for script in sync prepare build postinst finalize postoutput clean
```
Comment (DaanDeMeyer) [inline .github/workflows/ci.yml:170]: This overrides the existing `mkosi.postinst` script no?

Comment (behrmann) [reply .github/workflows/ci.yml]: And the prepare, too, dropped both from the list.

**Review by DaanDeMeyer — APPROVED** [review]

**Review by DaanDeMeyer — CHANGES_REQUESTED** [review]
Actually can you check if each script already exists and fail the CI if it does? Otherwise someone's going to run into some hard to debug issue when they try to add a script for the default image.

**Review by DaanDeMeyer — APPROVED** [review]

**Pull request merged**